recipes.js

```diff
@@ -1,11 +1,65 @@
-const mongoose = require('mongoose');
-const Schema   = mongoose.Schema;
-const data = require('./data.js')
+const chalk = require("chalk");
 
-mongoose.connect('mongodb://localhost/recipeApp')
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+const data = require("./data.js");
+
+const express = require("express");
+//const app = express;
+//app.set("view engine", "hbs");
+
+const Recipe = require("./recipes-model.js");
```

Comment:
good job, it is good practice to create your schema and your queries in separate files, you have linked them correctly!